stubs/emcache/__init__.py

```diff
@@ -0,0 +1,46 @@
+from typing import List
```

Comment:
maybe we should contribute to emcache to provide types?

Reply:
Actually, it does provide types!
Let me remove this and check why mypy was barking